### PR TITLE
feat: Adds the ability to see the popular articles

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

## Description

This PR adds the popular article section to the main screen.

Fixes https://linear.app/chatwoot/issue/CW-2616/implement-popular-articles-section-in-the-main-page

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

**Screenshot**
image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by nithindavid — CHANGES_REQUESTED** [review]
Looks good overall, please see the minor comments.

File: app/views/public/api/v1/portals/_featured_articles.html.erb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<% featured_articles = articles.where(status: :published).order_by_views.take(6) %>
+<% if featured_articles.count >= 6 %>
```
[the remark this answers is not in the document]
Comment (iamsivin) [reply app/views/public/api/v1/portals/_featured_articles.html.erb:2]: @nithindavid We will not show the featured articles section if there are less than six articles.

[the remark this answers is not in the document]
Comment (nithindavid) [reply app/views/public/api/v1/portals/_featured_articles.html.erb:2]: The check is the opposite right?

[the remark this answers is not in the document]
Comment (scmmishra) [reply app/views/public/api/v1/portals/_featured_articles.html.erb:2]: @nithindavid is right here, we should show whatever is available :)

**Pull request merged**